app/controllers/idv/review_controller.rb

```diff
@@ -81,6 +82,23 @@ def log_reproof_event
       irs_attempts_api_tracker.idv_reproof
     end
 
+    def log_irs_tmx_fraud_check_event
```

Comment:
```suggestion
   def log_irs_tmx_fraud_check_event
      return unless FeatureManagement.proofing_device_profiling_collecting_enabled?
      threatmetrix_review_pending = !idv_session.profile.active && idv_session.profile.threatmetrix_review_pending?
      failure_reason = { deactivation_reason: ['threatmetrix_review_pending'] } if threatmetrix_review_pending
      irs_attempts_api_tracker.idv_tmx_fraud_check(
        success: !threatmetrix_review_pending,
        failure_reason: failure_reason,
      )
    end
```

Reply:
@zachmargolis maybe this way?

Reply:
I was more concerned about which file it goes in